docs/data/sdks/typescript-browser/index.md

```diff
@@ -34,17 +34,20 @@ Use [this quickstart guide](../sdk-quickstart#browser) to get started with Ampli
 ### Configuration
 
 --8<-- "includes/sdk-ts-browser/shared-configurations.md"
+    |`attribution`| `AttributionOptions`. Configurations for web attribution plugin | Check "Attribution Options" config table below |
+    |`defaultTracking`| `boolean | DefaultTrackingOptions`. Configurations for default event tracking | Check [tracking default events](./#tracking-default-events)|
 
 In addition to the basic configuration options, there also has options to configure attribution.
     
 ???config "Attribution Options"
     |<div class="big-column">Name</div>| Description| Default Value|
     |---|----|---|
-    |`attribution.disabled`| `boolean`. Whether disable the attribution tracking. | `false` |
-    |`attribution.excludeReferrers`| `string[]`. Exclude the attribution tracking for the provided referrers string | Including all referrers by default. |
-    |`attribution.initialEmptyValue`| `string`. Customize the initial empty value for attribution related user properties to any string value. | `EMPTY` |
-    |`attribution.trackNewCampaigns`| `boolean`. Whether tracking new campaigns on the current session. | `false` | 
-    |`attribution.trackPageViews`| `boolean`. Whether track page view on attribution. | `false` |
+    |`config.attribution.disabled`| `boolean`. Whether disable the attribution tracking. | `false` |
+    |`config.attribution.excludeReferrers`| `string[]`. Exclude the attribution tracking for the provided referrers string | Including all referrers by default. |
+    |`config.attribution.initialEmptyValue`| `string`. Customize the initial empty value for attribution related user properties to any string value. | `EMPTY` |
+    |`config.attribution.resetSessionOnNewCampaign`| `boolean`. Whether to reset user sessions when a new campaign is detected. Note a new| `false` |
+    |`config.attribution.trackNewCampaigns`| `boolean`. Whether tracking new campaigns on the current session. | `false` | 
+    |`config.attribution.trackPageViews`| `boolean`. Whether track page view on attribution. | `false` |
```

Comment:
Maybe we should add a callout here about DET taking precedence and link to the section below.

Reply:
Why we added `config.` here?

Reply:
Hi @yuhao900914, I feel like adding `config` here can make the hierarchy more clear. Also added `config` to `config.defaultTracking`